source/agora/utils/Utility.d

```diff
@@ -227,3 +227,47 @@ public struct UbyteHexString
         });
     }
 }
+
+/// Utility class to implement vibe.d ResultSerializer.serialize and
+/// ResultSerializer.serialize templates
+public class ResultSerializerUtil
```

Comment:
It's unusual in our code base to use `class` for namespacing (but I think we do so in a couple places).
What's the motivation ?